tools/releases/dockerfiles/Dockerfile.kuma-init

```diff
@@ -1,6 +1,29 @@
-FROM docker.io/istio/proxy_init:1.1.17
+FROM ubuntu:bionic as builder
```

Comment:
Interesting, why not `golang:1.14.4-alpine3.12`?

Reply:
Discussed to stick with ubuntu:bionic for now